adaptive_scheduler/scheduler.py

```diff
@@ -699,6 +702,8 @@ def run_once(self):
             # We should force a rerun in any case in case the network events and requests haven't changed
             rerun_required = True
             self.log.warning("Skipping Scheduling Run: {}".format(repr(eee)))
+        # Reset the warm starts flag back to the input setting at the end of each run
```

Comment:
If the except clause on line 700 is entered, do we still want to set the warm start setting back to the original value? Like in the case when `create_new_schedule()` errors out and a new schedule was not able to be generated or something, would it be a problem if the next run had a different warm start setting?